programs/cleos/main.cpp

```diff
@@ -2374,17 +2412,23 @@ int main( int argc, char** argv ) {
         EOS_ASSERT( fc::exists( abiPath ), abi_file_not_found, "no abi file found ${f}", ("f", abiPath)  );
 
         abi_bytes = fc::raw::pack(fc::json::from_file(abiPath).as<abi_def>());
-
+        if (abi_bytes.size() > 0 && abi_bytes.size() == old_abi.size()) {
+           duplicate = (memcmp(&(abi_bytes[0]), &(old_abi[0]), abi_bytes.size()) == 0);
+        }
       } else {
         abi_bytes = bytes();
```

Comment:
same comment about checking for "duplicate" clears

Reply:
sure will fix that.